main.go

```diff
@@ -472,6 +502,29 @@ func flashHexUsingMSD(volume, tmppath string) error {
 	return moveFile(tmppath, d[0]+"/flash.hex")
 }
 
+func windowsFindUSBDrive(volume string) (string, error) {
+	cmd := exec.Command("wmic",
+		"PATH", "Win32_LogicalDisk", "WHERE", "VolumeName = '"+volume+"'",
```

Comment:
There is no sanity-checking of the volume name. That's not a problem right now, but may be a problem if we want to accept untrusted volume names in the future.

Reply:
We can add that in a future PR.